pb/transaction.proto

```diff
@@ -58,7 +58,7 @@ message Coinbase {
 }
 
 message SigChainTxn {
-	bytes sigChain     = 1;
```

Comment:
Will it cause data incompatibility, if some nodes serialize by the old definition?

Reply:
It won't because protobuf convert snake_case to camelCase automatically for go. Also, even we change the field name completely, the data will still be compatible because protobuf only use the field id (`= 1` part) for serialization/deserialization.